docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.7'
+services:
+# DATA_BASE**********************************
+  postgres:
+    image: postgres:latest
+    network_mode: bridge
+    container_name: quizter_db
+    expose:
+    - 5432
+    ports:
+      - 5432:5432
+    environment:
+         - POSTGRES_PASSWORD=password
+         - POSTGRES_USER=postgres
+         - POSTGRES_DB=quizter
+    restart: unless-stopped
+# APP*****************************************
+  quizter_app:
+    image: quizter_app:latest
+    network_mode: bridge
+    container_name: quizter_app
+    expose:
```

Comment:
Let's add `spring.datasource.url=jdbc:postgresql://postgres:5432/quizter` to env section for quzter_app container. It will override default in `application-dev.properties`, so we don't need to modify original file to make it work with docke